common/src/main/java/journeymap/api/v2/client/display/Overlay.java

```diff
@@ -297,8 +297,18 @@ public boolean isActiveIn(UIState uiState)
     {
         return ((uiState.active && this.dimension == uiState.dimension)
                 && activeUIs.contains(uiState.ui)
-                && activeMapTypes.contains(uiState.mapType)
-                && (this.minZoom <= uiState.zoom && this.maxZoom >= uiState.zoom));
+                && activeMapTypes.contains(uiState.mapType));
+    }
+
+    /**
+     * Whether the zoom of the given context is within the zoom range of the overlay.
+     *
+     * @param uiState    UIState
+     * @return true if the zoom of uiState is within the zoom range of the overlay
+     */
+    public boolean isInZoomRange(UIState uiState)
+    {
+        return (this.minZoom <= uiState.zoom && this.maxZoom >= uiState.zoom);
```

Comment:
I separated the zoom check so I could fix a bug later in Journeymap.

Reply:
nice